doc/tutorial/tutorial.md

```diff
@@ -1221,22 +1221,45 @@ vec.filter(|v| v.len() > 1)   // [[1,2,3], [4,5,6]]
 #### Negations and antijoins
 
 Let's assume we want to compute a relation `SanitizedEndpoint`
-containing the endpoints that do not appear in a `Blacklisted`
+containing the endpoints that do not appear in a `Blocklisted`
 relation:
 
 ```
-input relation Blacklisted(ep: string)
+input relation Blocklisted(ep: string)
 output relation SanitizedEndpoint(ep: string)
 
 SanitizedEndpoint(endpoint) :-
     EndpointString(endpoint),
-    not Blacklisted(endpoint).
+    not Blocklisted(endpoint).
 ```
 
 The `not` operator in the last line eliminates all endpoint values
-that appear in `Blacklisted`.  In database terminology this is known
+that appear in `Blocklisted`.  In database terminology this is known
 as *antijoin*.
 
+#### Left join
+
+A *left join* includes all of the records in one relation plus fields
+from matching records from a second relation.  In DDlog, we can build
+left joins using negation.  For example:
+
+```
+output relation EndpointSanitization(ep: string, sanitized: bool)
```

Comment:
Thanks for improving the docs!

To someone who jumped straight to the Left join section, it may not be obvious that there is a negation hiding inside `SanitizedEndpoint`. It may be a good idea to either point the reader to the definition of `SanitizedEndpoint` above or just inline its definition here:

```
EndpointSanitization(endpoint, true) :- 
    EndpointString(endpoint),
    not Blocklisted(endpoint).
```

Reply:
OK, I pushed an additional commit that does that. I don't know whether you'd prefer me to squash the new commit; let me know if so.